### PR TITLE
Fix directory for coverage files

### PR DESCRIPTION
**Pull request by azboss2021**
**_WHAT DOES THIS PR DO?_**

  - This PR changes the directory in the .yaml file so that github actions can successfully push the code coverage report to CodeCov

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

  - #14 

**_HOW DO I TEST OUT THIS PR?_**

  - Push a commit that changes any of the files related to src code and it will test and push the results to CodeCov

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -81,7 +81,7 @@ jobs:
         env:
           CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
         with:
-          directory: web/cobertura-coverage.xml
+          directory: web/coverage/cobertura-coverage.xml
```
Comment (danielogen) [inline .github/workflows/ci.yaml:84]: Check out comment in PR #51

**Review by danielogen — CHANGES_REQUESTED** [review]
Requesting minor changes

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (azboss2021) [reply .github/workflows/ci.yaml]: Completed

**Comment by danielogen** [comment]
@azboss2021 , We can close this PR without merge since the changes are in PR #51

**Review by carrec-2 — APPROVED** [review]

**Pull request closed without merging**